recipes/ucc/meta.yaml

```
@@ -0,0 +1,53 @@
+package:
+  name: ucc
+  version: v1.2.0
+
+source:
+  url: https://github.com/openucx/ucc/archive/refs/tags/v1.2.0.tar.gz
```

Would template the version throughout to make sure that when the package version is bumped the source URL is updated accordingly. Otherwise one could end up bumping the version and using old source code to build the package

Note: Am dropping the `v` from the `package/version` field in this suggestion. This is easier to work with in various build tooling around recipes. Plus the `v` isn't communicating anything we won't already know

```suggestion
{% set version = "1.2.0" %}

package:
  name: ucc
  version: {{ version }}

source:
  url: https://github.com/openucx/ucc/archive/refs/tags/v{{ version }}.tar.gz
```